docs/tsconfig.json

```diff
@@ -9,8 +9,16 @@
     "skipLibCheck": true,
     "esModuleInterop": true,
     "incremental": true,
-    "types": ["@mui/internal-test-utils/initMatchers", "chai-dom", "mocha"]
+    "types": ["@mui/internal-test-utils/initMatchers", "vitest/globals"]
```

Comment:
Just to note that this will add the globals to all docs code, even non-tests. Ideally we have separate tsconfigs for tests and docs, perhaps explicitly importing them in test files could already remove a bit of this friction?

Reply:
:+1: I'm also for importing them explicitly